examples/02-plot/chart_basics.py

```diff
@@ -0,0 +1,129 @@
+"""
+Chart Basics
+~~~~~~~~~~~~
+
+
+This example shows how different types of charts can be added to the scene. A more complex example, showing how to
+combine multiple charts as overlays in the same renderer, is given in `chart_overlays`.
+"""
+
+import pyvista as pv
+import numpy as np
+RNG = np.random.default_rng(1)
```

Comment:
Let's avoid having all cap variables.  These are generally used for [constants](https://www.python.org/dev/peps/pep-0008/#constants)

Reply:
It's also numpy custom to just call this `rng`. I support use of the new random API!

Reply:
ok, I will change that. Seeding was mainly to get some consistent plots while trying out things, but can definitely be removed in the final version.

Reply:
Actually, I think seeding it is fine.  In `numpydoc` with their plot directive, they always seed their random examples to generate consistent plots.  I think we should do the same.

Reply:
There's a trade-off between reproducibility and best practices. I tend to prefer setting a good example unless there's a very good reason not to. Not teaching users that seeding generators is normal is important. Getting the exact same plot is not that important to me as long as the overall look is the same and there are no edge cases that can break the example.

This is just to explain my stance. We can go ahead and seed regardless.

Reply:
I'm either way for seeding.  @dcbr, I'll leave it up to you.

Reply:
Ok, I can understand both arguments pro and con. My personal view is slightly more in favour of seeding though. In these examples, I see the rng calls more as a shorthand notation to quickly get some interesting data instead of having to hardcode some arbitrary datasets yourself. The actual randomness/uniqueness of this generated data is then less important than the consistency imo. I could add a short comment to the example to inform the user why seeding is used:
`rng = np.random.default_rng(1)  # Seeded random number generator for consistent data generation`